CMSgov/design-system
# [WNMGDS-841] Update scripts to allow for single test or folder to run

**Note:** 
- Add the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) to the PR title like this `[WNMGDS-10] - title of pr here` to link to the related issue in Jira.
- You can automatically [close related GitHub issues by using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- A demo doc site is automatically generated for this branch at `http://design-system-demo.s3-website-us-east-1.amazonaws.com/{branch_name}`. For example, the demo for `WNMGDS-10/feature` will be located at `http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-10/feature`.
- If your changes involve code please update the snapshots by running `yarn update-snapshots`.


**Please follow the format below and remove any sections that aren't relevant.**
## Summary
The only option before was to run all unit tests instead of just the test you are working on, which delays development.

### Changed
- Updated cmsds scripts to look for a filePath for the test to run

## How to test
- Pull locally and run:
- `yarn run test:unit` - To make sure the full test suite still runs
- `yarn run test:unit ./packages/design-system/src/components/ChoiceList` to make sure you can run all tests in a single directory
- `yarn run test:unit ./packages/design-system/src/components/ChoiceList/ChoiceList.test.jsx` to make sure you can run all tests in a single file

You can try other files other than ChoiceList too
